PackageSamples/Packages/D/D.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <NuspecFile>D.nuspec</NuspecFile>
+
+	<GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);SetNuspecProperties</GenerateNuspecDependsOn>
+  </PropertyGroup>
+
+<Target Name="SetNuspecProperties">
+  <PropertyGroup>
+    <NuspecProperties>$(NuspecProperties);Configuration=$(Configuration)</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);version=$(InformationalVersion)</NuspecProperties>
+    <NuspecProperties>$(NuspecProperties);OutDir=$(OutputPath)</NuspecProperties>
+  </PropertyGroup>
+</Target>
+	<ItemGroup>
```

Comment:
The mixture of indenting in all of these projects seems odd, worth fixing?

Reply:
Fixed.